src/html/HTMLWriter.jl

```diff
@@ -850,13 +862,30 @@ function render_head(ctx, navnode)
     head(
         meta[:charset=>"UTF-8"],
         meta[:name => "viewport", :content => "width=device-width, initial-scale=1.0"],
+
+        # Title tag and meta tags
         title(page_title),
+        meta[:name => "title", :content => page_title],
+        meta[:property => "og:title", :content => page_title],
+        meta[:property => "twitter:title", :content => page_title],
+
+        # Description meta tags
+        meta[:name => "description", :content => description],
+        meta[:property => "og:description", :content => description],
+        meta[:property => "twitter:description", :content => description],
+
+        # Canonical URL tags
+        isnothing(canonical) ? empty_node() : meta[:property => "og:url", :content => canonical],
+        isnothing(canonical) ? empty_node() : meta[:property => "twitter:url", :content => canonical],
+        isnothing(canonical) ? empty_node() : link[:rel => "canonical", :href => canonical],
```

Comment:
One option here would be to assign these tags into a 

```
meta_urls = isnothing(canonical) ? [VOID] : [
    meta[...], ...
]
```

and them just splat that here. Either way is fine though.